site/features.md

```diff
@@ -6,44 +6,70 @@ menu_order: 15
-    * [Powerful Search](#powerful-search)
+ * [Powerful Search](#powerful-search)
  * [Real-time App and Container Metrics](#real-time-app-and-container-metrics)
- * [Interact With and Manage Containers](#interact-with-and-manage-containers)
- * [Troubleshoot Apps](#troubleshoot-apps)
-
+ * [Troubleshoot and Manage Containers](#interact-with-and-manage-containers)
+ * [Generate Custom Metrics using the Plugin API](#custom-plugins)
+ 
 
 ##<a name="topology-mapping"></a>Topology Mapping
 
-Scope builds logical topologies of your application and infrastructure.  A topology is a collection of nodes and edges, where nodes can represent objects like processes, container or hosts.  In Scope, edges represent TCP connections between nodes.  Scope displays an intelligent map of your app, so that different node types can be easily viewed and specific microservices can be drilled down on and inspected.
+Scope builds logical topologies of your application and infrastructure.  A topology is a collection of nodes and edges, where nodes represent objects like processes, container or hosts.  In Scope, edges indicate TCP connections between nodes.  Different node types can be filtered on and viewed and specific microservices can be drilled down on and inspected. Nodes are presented in a particular order with the clients above servers. As a general rule, you can read a Weave Scope view by going from top to bottom.
+
+!['Automatic Topology Mapping in Graph Mode'](images/topology-map.png)
+
+##<a name="views"></a>Views in Scope
+
+Views in Scope can be thought of as high-level filters on your containerized microservices as they run in the cloud. They are categorized into: Processes, Containers, Orchestrators, and Hosts. The Orchestrator view is context-sensitive, and as a result, if an app is running in Kubernetes, then Scope displays Pods, Replica Sets, Deployments, and Services that allow you to drilldown on Kubernetes clusters. 
+
+!['ECS Views in Scope'](images/kubernetes-view.png)
+
+If your app is deployed to the Amazon EC2 cluster service, then Scope displays Services and Tasks that make up an ECS cluster.
+
+!['ECS Views in Scope'](images/ECS-view-tasks.png)
+
+For information on how to install Scope on Kubernetes see, "[Installing Scope on Kubernetes](installing.md#k8s)
+
+For information on how to install Scope on the AWS EC2 Container Service see, "[Installing Scope on Amazon ECS](installing.md#ecs)
+
+Under the hosts view, and if you are running Weave Net for container networking a specific Weave Net view appears. This view is useful for troubleshooting any networking problems you may be having. This view displays a number of Weave Net specific attributes such as whether quorum has been reached, the IP addresses used, whether fast datapath is enabled, or if encryption is running and many other useful attributes. See [Weave Net User Guide](https://www.weave.works/docs/net/latest/features/) for more information. 
```

Comment:
missing comma after networking _and if you are running Weave Net for container networking, a specific Weave Net view appears_